pom.xml

```diff
@@ -199,6 +210,7 @@ and
       <groupId>com.google.inject</groupId>
       <artifactId>guice</artifactId>
       <version>6.0.0</version>
+      <classifier>classes</classifier>
```

Comment:
I can't quite understand what this is for (I have never even used `<classifier>classes</classifier>` before), and tests seemed to work fine when I removed it. Is it better to keep it for some particular reason?

Reply:
This is "guice w/o asm shaded in"